fetch.bs

```diff
@@ -3049,6 +3055,17 @@ in addition to <a>HTTP fetch</a> above.
  to `<code>GET</code>` and <var>request</var>'s <a for=request>body</a>
  to null.
 
+ <li>
+  <p>If <var>request</var>'s <a for=request>body</a> is non-null, then set <var>request</var>'s
+  <a for=request>body</a> to the first part of <a lt=extract for=BodyInit>extracting</a>
+  <var>request</var>'s <a for=request>body</a>'s <a for=body>source</a>.
```

Comment:
I'm still a little confused by the new proposed setup. Now body can have a source, but body itself can also be a byte sequence? I'm not sure that makes sense.